app/src/main/res/layout/sheet_link_id_with_patient.xml

```diff
@@ -32,28 +31,26 @@
     android:layout_marginTop="@dimen/spacing_16"
     android:orientation="horizontal">
 
-    <org.simple.clinic.widgets.PrimarySolidButton
+    <Button
       android:id="@+id/linkidwithpatient_button_cancel"
-      style="@style/Clinic.V2.Button.Solid.WithoutIcon.Blue1"
       android:layout_width="0dp"
       android:layout_height="wrap_content"
       android:layout_marginEnd="@dimen/spacing_8"
       android:layout_weight="1"
       android:background="@drawable/button_outline_ripple"
       android:backgroundTint="@color/grey1"
       android:text="@string/linkidwithpatient_button_cancel"
-      android:textAppearance="@style/Clinic.V2.TextAppearance.Button1.Grey1" />
+      android:textAppearance="@style/Clinic.V2.TextAppearance.Button1.Grey1"
+      tools:ignore="ButtonStyle" />
 
-    <org.simple.clinic.widgets.PrimarySolidButton
+    <Button
```

Comment:
This button is not the same size as in the design. Please fix?

Reply:
Fixed.